server/server.mjs

```diff
@@ -9,22 +9,45 @@ const port = process.env.PORT || 4000;
 // items defines the routes
 const items = express.Router();
 
+// gets all items
 items.get("/", async (request, response) => {
   const items = await db.getItems();
   console.log("hello test test");
-  response.json(items);
+  response.status(201).json(items);
```

Comment:
A successful Get request usually returns a status of `200`. I like to use this as a refresher https://developer.mozilla.org/en-US/docs/Web/HTTP/Status#successful_responses